pkg/inframanager/p4/service_es2k.go

```diff
@@ -19,483 +19,233 @@ package p4
 import (
 	"context"
 	"fmt"
+
 	"github.com/antoninbas/p4runtime-go-client/pkg/client"
 	"github.com/ipdk-io/k8s-infra-offload/pkg/inframanager/store"
-	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
-	log "github.com/sirupsen/logrus"
+	"github.com/ipdk-io/k8s-infra-offload/pkg/types"
+
+	//p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
 	"net"
+
+	log "github.com/sirupsen/logrus"
 )
 
-func WriteDestIpTable(ctx context.Context, p4RtC *client.Client,
-	podIpAddr []string, portID []uint16, modBlobPtrDnat []uint32,
-	action InterfaceType) error {
+var (
+	service_table_names = []string{"k8s_dp_control.ipv4_to_port_table_tx_tcp",
+		"k8s_dp_control.ipv4_to_port_table_tx",
+		"k8s_dp_control.write_dest_ip_table",
+		"k8s_dp_control.write_source_ip_table",
+		"k8s_dp_control.rx_src_ip",
+		"k8s_dp_control.tx_balance"}
+
+	service_action_names = []string{"k8s_dp_control.set_vip_flag_tcp",
+		"k8s_dp_control.set_vip_flag",
+		"k8s_dp_control.update_dst_ip_mac",
+		"k8s_dp_control.update_src_ip_mac",
+		"k8s_dp_control.set_source_ip",
+		"k8s_dp_control.set_default_lb_dest"}
+)
```

Comment:
Same comment as for cni_table_names and cni_action_names.

Reply:
Same comment as above. Will take it as a future enhancement.